StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,109 @@
+using Mars_Luiz.Pages;
+using OpenQA.Selenium.Chrome;
+using System;
+using TechTalk.SpecFlow;
+using Mars_Luiz.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using NUnit.Framework.Legacy;
+
+namespace Mars_Luiz.StepDefinitions
+{
+    [Binding]
+    public class LanguageStepDefinitions :CommonDriver
+    {
+        [Given(@"I logged into the portal successfull")]
+        public void GivenILoggedIntoThePortalSuccessfull()
+        {
+            //Open chrome browser
+            driver = new ChromeDriver();
+            driver.Manage().Window.Maximize();
+            //Sign IN page object initialization and definition
+            SignInPage signInPageObj = new SignInPage();
+            signInPageObj.SignInSteps(driver);            
+        }
+
+        [When(@"I navigate to Profile page")]
+        public void WhenINavigateToProfilePage()
+        {
+            //Profile page object initialization and definition
+            ProfilePage homePageObj = new ProfilePage();
+            homePageObj.GoToProfilePage(driver);
+        }
+
+     // Add Language
+
+        [When(@"I add the Language in the Language section")]
+        public void WhenIaddTheLanguageToLanguangePage()
+        {
+            LanguagePage profilePageObj = new LanguagePage();
+            profilePageObj.Addlanguage(driver);
```

Comment:
Please do not pass the driver In action methods (methods which in pages doing things like create, delete etc ) instead get the pages to extend to common driver where you will create a common driver class which will initialise the web driver